test/smoke_test/smoke_test.py

```diff
@@ -3,29 +3,78 @@
 import torch
 import torchvision
 import torchaudio
+import pathlib
```

Comment:
Please define SCRIPT_DIR global variable and use it to resolve path to object relative to script
```suggestion
from pathlib import Path

SCRIPT_DIR = Path(__file__).parent
```